File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { UuidGeneratorApi2SDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await UuidGeneratorApi2SDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- UuidGeneratorApi2 SDK exists test

local sdk = require("uuid-generator-api2_sdk")

describe("UuidGeneratorApi2SDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
